oncotator/index/TabixIndexer.py

```diff
@@ -100,7 +100,8 @@ def index(destDir, inputFilename, fileColumnNumList=None, preset=None):
             in the tsv file. Typically, this would be [chr,start,end]  or [gene, startAA, endAA]
```

Comment:
I'm a bit confused about this method in general actually, it claims to tabix index a file, and gzip it if it's not already.  It looks like it creates a new index if the file is not gzipped.  It copies the file but it doesn't seem gzip it.  Then it returns the index file.    If it is gzipped then copies the file, copies over the existing index, explodes if there isn't one, and then returns the copied file, not the index file.  It seems like kind strange behavior.  Is that what's intended?